apps/auracast.py

```diff
@@ -965,6 +965,11 @@ async def run_transmit(
                 bumble.device.IsoPacketStream(big.bis_links[1], 64),
             ]
 
+            for queue in iso_queues:
```

Comment:
Thanks for catching that. Somehow this got accidentally removed in a recent PR and went unnoticed (it still worked with the BT controller on my Pixel phone...).
However, it might be more straightforward to express this as:
```
for bis_link in big.bis_links:
    await bis_link.setup_data_path(
        direction=bis_link.Direction.HOST_TO_CONTROLLER
)
```
above line 963